ENIGMAsystem/SHELL/Graphics_Systems/Direct3D11/DX11screen.cpp

```diff
@@ -52,22 +52,20 @@ void screen_init()
   enigma::gui_width = window_get_region_width();
   enigma::gui_height = window_get_region_height();
 
-  //d3dmgr->Clear(0, NULL, D3DCLEAR_TARGET, D3DCOLOR_XRGB(0, 0, 0), 1.0f, 0);
-  //d3dmgr->Clear(0, NULL, D3DCLEAR_ZBUFFER, D3DCOLOR_XRGB(0, 0, 0), 1.0f, 0);
+  m_deviceContext->ClearDepthStencilView(m_depthStencilView, D3D11_CLEAR_DEPTH, 1.0f, 0);
 
-  if (!view_enabled) {
-  //d3dmgr->Clear(0, NULL, D3DCLEAR_TARGET, D3DCOLOR_XRGB(0, 0, 0), 1.0f, 0);
-
-  screen_set_viewport(0, 0, window_get_region_width(), window_get_region_height());
-  d3d_set_projection_ortho(0, 0, window_get_region_width(), window_get_region_height(), 0);
+  if (!view_enabled)
```

Comment:
This is extremely general code... didn't cheeseboy just extract all this? Or is he still in the process?

Reply:
That was me who just deduplicated `screen_redraw`: 8424fa6fa917da179d2a20d2031441ca68501868

`screen_init` does need deduplicated, but I am saving that for another pr, I will discuss it with you at that time.

Reply:
I plan on looking at it soon, but for now I've #1337.

Reply:
I feel as though #1337 should have been more special. Oh well.